src/components/RegistrationDetailsPage.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import { withRouter } from 'react-router-dom'
+import { connect } from 'react-redux'
+import ReactDragList from 'react-drag-list'
+
+import registrationActions from '../reducers/actions/registrationActions'
+
+import Typography from '@material-ui/core/Typography'
+import { Input, Card, CardContent, Select, MenuItem } from '@material-ui/core'
+import TopicDialog from './TopicDialog'
+
+import './RegistrationDetailsPage.css'
+
+class RegistrationDetailsPage extends React.Component {
+  render() {
+    const registration = this.props.ownRegistration
+    const { student, preferred_topics, questions, createdAt } = registration
+    const { first_names, last_name, student_number, email } = student
+
+    let firstname = ''
+    if (first_names.includes('*')) {
+      firstname = first_names.split('*')[1]
+    }
+    firstname = firstname.split(' ')[0]
+
+    const submittedDate = createdAt.replace(/T/, ' ').substr(0, 16)
```

Comment:
Do I understand correctly that this is used to format `"2019-01-01T10:00:00.000Z"` --> `""2019-01-01 10:00"`?

Sequelize seems to be producing/mapping/getting the dates from PostgresSQL in UTC time. This will probably produce the timestamp in UTC time, since `JSON.stringify` uses `Date.toISOString()`.

We should probably parse the timestamp string and format it afterwards. `new Date(timestamp)` seems to produce a `Date` with the user's timezone, at least on Chrome. The internet does tell me though that browser support for parsing ISO 8601 timestamps (.toISOString()) via `new Date()`/`Date.parse` is not great...do we care about older browsers though?

After parsing the date, we could just use `.toLocaleString()` to get a timestamp like this:
```
> new Date().toLocaleString('fi-FI')
// "5.2.2019 klo 19.06.04"
```
or use `date-fns` (which is way lighter and more modular than `momentjs`) for more control over the format:

```
$ npm i -s date-fns
```
then
```js
import formatDate from 'date-fns/format'
// ...
const formatSubmitDate = createdAt => {
  const date = new Date(createdAt)
  return formatDate(date, 'D.M.YYYY HH:mm')
}
```
```
> formatSubmitDate("2019-02-05T17:08:19.453Z")
// "5.2.2019 19:08"
```
Notice how the hour changes from 17 (UTC) to 19 (GMT+2/Helsinki)

Reply:
Oops. I forgot the UTC aspect totally. I must look into information you provided and refactor this someway. Using a new module to parse datetimes seems a bit like popping more node.js painkillers though...

Reply:
Yeah it kinda is. `toLocaleString('fi-FI')` is probably good enough.